src/canvas-extractor/README.md

```diff
@@ -82,11 +81,18 @@ data and would typically span a semester or equivalent school calendar timespan.
 * ERROR
 * CRITICAL
 
-\*** It is allowed to add more than one value, valid options for _feature_:
-* activities
-* attendance
-* assignments
-* grades
+\*** Available features (can combine two or more):
+
+* activities (Enables the extraction of section activities and system activities)
+* attendance (Enables the extraction of attendance events)
+* assignments (Enables the extraction of assignments and submissions)
+* grades (Enables the extraction of grades)
+
+When setting features via `.env` file or through environment variable, combine features by using a bracketed comma-separate list, e.g.
+
+```none
+FEATURE=[activities, attendance, assignments, grades]
+```
```

Comment:
Can you apply the two updates above (line 71, lines 84-95) in the other two readmes as well?

Reply:
✔️